html_01/62134_Dimitar_Lazarov.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
This file is part of the PR from the first task: https://github.com/DimitarL/WEB2020-php/pull/1 It should not be part of this PR. Please delete the file from this PR.